setup.py

```diff
@@ -74,6 +75,7 @@ def main():
         "eth-typing",
         "eth-utils",
         "flake8",
+        "grpc",
```

Comment:
Will this cause issues since grpcio is included in the packages to be installed? cc  @fengtality @cardosofede. Mighht be wrong thou

Reply:
figured their difference please ignore

Reply:
I figured this has been addressed on https://github.com/hummingbot/hummingbot/pull/7214#issuecomment-2339935496